environment-dev.yml

```diff
@@ -2,7 +2,7 @@ name: modin
 channels:
   - conda-forge
 dependencies:
-  - pandas==2.0.2
+  - pandas>1,<2.1
```

Comment:
```suggestion
  - pandas>=2,<2.1
```

Reply:
This was my mistake :) In the original issue, Devin had written ">1.4,<1.6", and that works well when there's no pandas 2.0, but we need >= to require >= pandas 2